.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,47 @@
+name: 'Get name and newest digest for docker image'
+
+env:
+  REGISTRY: ghcr.io
+
+on:
+  workflow_call:
+    inputs:
+      tool:
+        description: 'Name of the tool'
+        required: false
+        default: 'tools'
+        type: string
+      sc_version:
+        description: 'SC Version tag'
+        required: false
```

Comment:
does this need a default value of 'main'?

Reply:
No runs successfully like that.